### PR TITLE
docs: add performance advice to disable default menu creation

### PR DESCRIPTION
**Pull request by bpasero**
#### Description of Change

Fixes https://github.com/electron/electron/issues/35512

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/main/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: docs: add performance advice to disable default menu creation

### CODE REVIEW

**Review by ckerr — APPROVED** [review]

File: docs/tutorial/performance.md [hunk 1]
```diff
@@ -432,3 +432,16 @@ As of writing this article, the popular choices include [Webpack][webpack],
 [parcel]: https://parceljs.org/
 [rollup]: https://rollupjs.org/
 [vscode-first-second]: https://www.youtube.com/watch?v=r0OeHRUCCb4
+
+
+### 8. Call `Menu.setApplicationMenu(null)` when you do not need a default menu
```
Comment (MarshallOfSound) [inline docs/tutorial/performance.md:437]: This block should be above the large block of links above

Comment (bpasero) [reply docs/tutorial/performance.md]: 👍

**Comment by bpasero** [comment]
This is good to go now, I made the text a bit more verbose and easier to read.

**Review by zcbenz — APPROVED** [review]

**Review by codebytere — APPROVED** [review]

**Comment by codebytere** [comment]
Lint failure:

```
docs/tutorial/performance.md:424:76 MD009/no-trailing-spaces Trailing spaces [Expected: 0; Actual: 1]
docs/tutorial/performance.md:430:74 MD009/no-trailing-spaces Trailing spaces [Expected: 0; Actual: 1]
```

**Comment by ckerr** [comment]
> Invoke-RestMethod : {"message":"Artifact not found or access denied."}

CI failure appears to be unrelated to the PR. Is anyone opposed to this being merged?

**Comment by nornagon** [comment]
It's a little surprising that this is so expensive! Is there any way we can work around this technically?

**Pull request merged**